File: src/main/java/plus/maa/backend/controller/request/user/PasswordResetVCodeDTO.java
package plus.maa.backend.controller.request.user;

import jakarta.validation.constraints.Email;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.experimental.Accessors;

/**
 * 通过邮件修改密码发送验证码请求
 */
@Data
@Accessors(chain = true)
@NoArgsConstructor
@AllArgsConstructor
public class PasswordResetVCodeDTO {
    /**
     * 邮箱
     */
    @Email(message = "邮箱格式错误")
    private String email;
}

File: src/main/kotlin/plus/maa/backend/controller/request/user/PasswordResetVCodeDTO.kt
package plus.maa.backend.controller.request.user

import jakarta.validation.constraints.Email
import jakarta.validation.constraints.NotBlank

/**
 * 通过邮件修改密码发送验证码请求
 */
data class PasswordResetVCodeDTO(
    /**
     * 邮箱
     */
    @field:NotBlank(message = "邮箱格式错误")
    @field:Email(message = "邮箱格式错误")
    val email: String,
)
